yoti-sdk-sandbox/src/test/java/com.yoti.api.client.sandbox/profile/request/YotiTokenRequestTest.java

```diff
@@ -78,8 +83,25 @@ public void shouldAddAnAttributeOnlyOnce() {
                 .withAttribute(SOME_ATTRIBUTE)
                 .build();
 
-        assertEquals(1, yotiTokenRequest.getSandboxAttributes().size());
-        assertEquals(SOME_ATTRIBUTE_NAME, yotiTokenRequest.getSandboxAttributes().get(0).getName());
+        assertThat(yotiTokenRequest.getSandboxAttributes(), hasSize(1));
+        assertThat(yotiTokenRequest.getSandboxAttributes(), hasItem(SOME_ATTRIBUTE));
+    }
+
+    @Test
+    public void shouldAllowSameAttributeWithDifferingDerivationNames() {
+        SandboxAttribute derivationAttribute = SandboxAttribute.builder()
+                .withName(SOME_ATTRIBUTE_NAME)
+                .withDerivation("derivation1")
+                .build();
+
+        YotiTokenRequest yotiTokenRequest = YotiTokenRequest.builder()
+                .withAttribute(SOME_ATTRIBUTE)
+                .withAttribute(derivationAttribute)
```

Comment:
Will the derived attributes have anchors?

Reply:
Yes.  My understanding is that they will come along with all the _anchors_ from the original _date_of_birth_ attribute.